source/math/FloatBox2D.ooc

```diff
@@ -80,9 +87,19 @@ FloatBox2D: cover {
 		This new(left, top, width, height)
 	}
 	contains: func (point: FloatPoint2D) -> Bool {
-		this left <= point x && point x < this right && this top <= point y && point y < this bottom
+		this left <= point x && point x <= this right && this top <= point y && point y <= this bottom
 	}
 	contains: func ~box (box: FloatBox2D) -> Bool { this intersection(box) == box }
+	contains: func ~FloatPoint2DVectorList (list: FloatPoint2DVectorList) -> VectorList<Bool> {
+		result := VectorList<Bool> new()
+		for (i in 0..list count) {
+			if (this contains(list[i]))
+				result add(true)
+			else
+				result add(false)
+		}
+		result
+	}
```

Comment:
I need the indices, not the actual values. So either I give back a list with indices or a list with booleans. And since the other contains functions give `Bool` back, I thought I do the same with this function. If you anyway think it's better to give back a list of indices, I change that.